pkg/client/tokenservice/file.go

```diff
@@ -14,35 +14,29 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package client
+package tokenservice
 
 import (
 	"encoding/binary"
 	"errors"
+	"github.com/codenotary/immudb/pkg/client/homedir"
 	"strings"
 )
 
-type TokenService interface {
-	SetToken(database string, token string) error
-	WithHds(hds HomedirService) TokenService
-	WithTokenFileName(tfn string) TokenService
-	IsTokenPresent() (bool, error)
-	DeleteToken() error
-	GetToken() (string, error)
-	GetDatabase() (string, error)
-}
-
-type tokenService struct {
+type file struct {
 	tokenFileName string
```

Comment:
file token service is not thread-safe where the in-memory service is thread-safe. Shouldn't we always guarantee thread-safety?